### ISSUE TITLE
Update Failed (not using Cloudflare or a security plugin)

### ISSUE THREAD

**Issue by moshemo** [WordPress/gutenberg#9675]
**Describe the bug**
I receive the Update Failed message (and can't save anything in Gutenberg).

**To Reproduce**
I simply go to Add New Post and try to create a new post.  After a second or so the message pops up.  I lose any work I do since it can't be saved.

**Expected behavior**

No error message and the ability to save/publish my work.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome Version 68.0.3440.106 (Official Build) (64-bit)

**Smartphone (please complete the following information):**

**Additional context**

I'm using version 3.7.0 of Gutenberg.  I am NOT using Cloudflare and I do NOT have a security plugin installed.  I also tried deactivating ALL other plugins and reverting back to the twenty seventeen theme.  Nothing helped.  I'm on cloudways hosting, but I am NOT using their CDN.  They are not aware of the problem.

I used the chrome dev tools and this is what shows up in network:

(anonymous) | @ | index.js?ver=1536258932:1
-- | -- | --
  | t | @ | index.js?ver=1536258932:1
  | b | @ | index.js?ver=1536258932:1
  | t | @ | index.js?ver=1536258932:1
  | m | @ | index.js?ver=1536258932:1
  | t | @ | index.js?ver=1536258932:1
  | (anonymous) | @ | index.js?ver=1536258932:1
  | t | @ | index.js?ver=1536258932:1
  | (anonymous) | @ | index.js?ver=1536258932:1
  | m | @ | index.js?ver=1536258932:1
  | (anonymous) | @ | index.js?ver=1536258932:1
  | t | @ | index.js?ver=1536258932:1
  | (anonymous) | @ | index.js?ver=1536258932:1
  | t | @ | index.js?ver=1536258932:1
  | S | @ | index.js?ver=1536258932:1
  | (anonymous) | @ | index.js?ver=1536258932:12
  | k | @ | index.js?ver=1536258932:12
  | (anonymous) | @ | index.js?ver=1536258932:12
  | e.(anonymous function) | @ | index.js?ver=1536258932:12
  | o | @ | index.js?ver=1536258932:12
  | s | @ | index.js?ver=1536258932:12
  | Promise.then (async) |   |  
  | o | @ | index.js?ver=1536258932:12
  | s | @ | index.js?ver=1536258932:12
  | (anonymous) | @ | index.js?ver=1536258932:12
  | t | @ | index.js?ver=1536258932:12
  | (anonymous) | @ | index.js?ver=1536258932:12
  | (anonymous) | @ | index.js?ver=1536258932:12
  | REQUEST_POST_UPDATE | @ | index.js?ver=1536258932:12
  | (anonymous) | @ | index.js?ver=1536258932:12
  | (anonymous) | @ | index.js?ver=1536258932:1
  | value | @ | index.js?ver=1536258932:1
  | (anonymous) | @ | index.js?ver=1536258932:12

Final note, if I deactivate Gutenberg, I can save a post with no problem.

Any idea how to fix this?

Thanks.

**Comment by moshemo** [comment]
It has resolved itself.  I don't know what happened.  I created a new user and went to the customizer.  It asked me to login again (which I did with the new users login details).  I then went back and reactivated gutenberg and created a post.  And lo and behold, problem fixed.

I think it says updated failed because I wasn't PROPERLY logged in (whatever that means -- I mean, I was logged in -- I could acivate and deactive plugins, create posts, etc).
